### PR TITLE
add storybooks for easier local dev on vscode UI

### PR DESCRIPTION
**Pull request by sqs**
Running `cd client/cody && pnpm run storybook` will open storybooks for the VS Code extension. There is a light shim in place to support rendering the VS Code extension UI with roughly the right theme colors.

### Storybook for `<App />` (VS Code main webview UI)

image
image

### Storybook for login component showing invalid state

image

## Test plan

n/a; dev only

### CODE REVIEW

File: client/cody/webviews/Header.module.css [hunk 1]
```diff
@@ -4,7 +4,7 @@
     font-family: var(--vscode-font-family);
     border-bottom: 1px solid var(--vscode-sideBarSectionHeader-border);
     padding: 1rem;
-    color: var(--vscode-sideBarTitle-foreground);
+    color: var(o--vscode-sideBarTitle-foreground);
```
Comment (thenamankumar) [inline client/cody/webviews/Header.module.css:7]: mistake?

File: client/cody/webviews/Login.story.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+import { Login } from './Login'
+import { VSCodeStoryDecorator } from './storybook/VSCodeStoryDecorator'
+
+const meta: ComponentMeta<typeof Login> = {
+    title: 'cody/Login',
+    component: Login,
+
+    decorators: [VSCodeStoryDecorator],
+
```
Comment (thenamankumar) [inline client/cody/webviews/Login.story.tsx:11]: new line not required, but not a blocker.

File: client/cody/webviews/Recipes.tsx [hunk 3]
```diff
@@ -16,19 +16,19 @@ export const recipesList = {
...
 
-export function Recipes(): React.ReactElement {
+export const Recipes: React.FunctionComponent<{ vscodeAPI: VSCodeWrapper }> = ({ vscodeAPI }) => {
     const onRecipeClick = (recipeID: string): void => {
         vscodeAPI.postMessage({ command: 'executeRecipe', recipe: recipeID })
     }
 
     return (
         <div className="inner-container">
             <div className="non-transcript-container">
```
Comment (thenamankumar) [inline client/cody/webviews/Recipes.tsx:26]: Is `inner-container` global? Why not move this as well to module.css

**Review by thenamankumar — APPROVED** [review]

File: client/cody/webviews/Recipes.tsx (hunk 3 above)
Comment (thenamankumar) [reply client/cody/webviews/Recipes.tsx:26]: yeah its global, we can keep it.

File: client/cody/webviews/Header.module.css (hunk 1 above)
Comment (thenamankumar) [reply client/cody/webviews/Header.module.css]: resolved it

File: client/cody/webviews/Login.story.tsx (hunk 2 above)
Comment (thenamankumar) [reply client/cody/webviews/Login.story.tsx]: resolved it

**Comment by sqs** [comment]
Thank you @thenamankumar!

**Pull request merged**